config/cachecfg.go

```diff
@@ -81,3 +81,47 @@ func (cCfg CacheCfg) AsTransCacheConfig() (tcCfg map[string]*ltcache.CacheConfig
 	}
 	return
 }
+
```

Comment:
since this is not something for the user to see, we can add these in an object instead of json string